packages/backend-core/src/users/db.ts

```diff
@@ -335,7 +336,7 @@ export class UserDB {
       }
       newUser.userGroups = groups || []
       newUsers.push(newUser)
-      if (isCreator(newUser)) {
+      if (await isCreator(newUser)) {
```

Comment:
Now, in pro submodule, `isCreator` requires a group role check, so it is an async funtion

Reply:
Backend-core is not using pro. This is using the code from shared-core, and as far as I can see, it is still using a sync call. Am I wrong?